indico/web/templates/forms/prefixed_text_widget.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'forms/base_widget.html' %}
+
+{% block html %}
+    <div class="i-form-field-fixed-width labeled input">
+        <span class="label">{{ prefix }}</span>
+        <input id="{{ field.id }}" name="{{ field.name }}" value="{{ field._value() }}" type="text">
```

Comment:
You need `{{ input_args|html_params }}` to forward any extra args (such as `disabled` or `required`) to the field.